nuxeo-features/nuxeo-automation/nuxeo-automation-jsf/src/main/java/org/nuxeo/ecm/automation/jsf/operations/AddErrorMessage.java

```diff
@@ -28,10 +28,11 @@
 /**
  * @author <a href="mailto:td@nuxeo.com">Thierry Delprat</a>
  */
-@Operation(id = AddErrorMessage.ID, category = Constants.CAT_UI, requires = Constants.SEAM_CONTEXT, label = "Add Error Message", description = "Add a message to be displayed in case the chain execution fails. This is a void operation - the input will be returned back as output", aliases = { "Seam.AddErrorMessage" })
+@Operation(id = AddErrorMessage.ID, category = Constants.CAT_UI, requires = Constants.SEAM_CONTEXT, label = "Add Error Message", description = "Add a message to be displayed in case the chain execution fails. This is a void operation - the input will be returned back as output", aliases = {
+        "WebUI.AddErrorMessage" })
```

Comment:
This added newline makes it ever so slightly more complex to review the diff, and is not strictly needed.